package.json

```diff
@@ -34,20 +34,20 @@
     "@vitest/ui": "^v2.0.0-beta.12",
     "c8": "7.12.0",
     "embedme": "1.22.1",
-    "eslint": "8.45.0",
-    "eslint-config-prettier": "8.8.0",
-    "eslint-config-typestrict": "1.0.5",
+    "eslint": "8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-typestrict": "^1.0.5",
```

Comment:
Are we comfortable moving to unpinned versions? I recall previous discussions where we had decided to pin them down for security reasons.

Reply:
I'm not anxious about it since these are dev dependencies but will happily defer to other opinions.

Reply:
I am also ok with unpinned versions

Reply:
good!